src/libraries/System.Text.Json/tests/System.Text.Json.FSharp.Tests/System.Text.Json.FSharp.Tests.fsproj

```
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="EnumTests.fs" />
     <Compile Include="Helpers.fs" />
```

File ordering is signfificant in F# projects, better move after the helpers module.
```suggestion
    <Compile Include="Helpers.fs" />
    <Compile Include="EnumTests.fs" />
```